### ISSUE TITLE
[Profile] Form control does not have a corresponding label

### ISSUE THREAD

**Issue by degraphic** [veshu/accessibility-bughunt-bluesource#79]
<h2>Steps to reproduce the behavior</h2>
1. Go to Wellmark app. <br>
2. Press Click PROFILE.<br>
3. Then Click  WAVE Web Accessibility Tool from Extention On Chrome

<h2>Actual behavior</h2>
We can see Form control does not have a corresponding label inside the form

<h2>Expected behavior</h2>
I think we can add Label inside the form to fix the error

**Device**: Laptop MacbookPro
**Operating System**: MacOs Mojave 10.14.6
**Browser(with version)**: Chrome Version 78.0.3904.70 (Official Build) (64-bit)

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non text content
**Tools used**: WAVE

**Screenshoot:**
<br>

![Screen Shot 2019-11-01 at 01 07 27](https://user-images.githubusercontent.com/4364214/67974376-e7e48680-fc44-11e9-989e-65a7f569fc8d.png)
<br>
![Screen Shot 2019-11-01 at 01 06 22](https://user-images.githubusercontent.com/4364214/67974378-e87d1d00-fc44-11e9-892e-4914a75b7a2c.png)
<br>
![device](https://user-images.githubusercontent.com/4364214/67978417-a2c45280-fc4c-11e9-8493-aa7f8ec507d0.png)
